.github/workflows/release-gh-notes.yml

```diff
@@ -0,0 +1,46 @@
+name: release-gh-notes
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        description: 'Specify the version for this release'
+        type: string
+      npm_package:
+        required: true
+        description: 'npm package of the release'
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: install npm package
+        run: npm install ${{ github.event.inputs.npm_package }}@${{ github.event.inputs.version }}
+        shell: bash
+       - name: Get manifest file
+         id: manifest
+         run: echo ::set-output name=manifest::$(cat node_modules/${{ github.event.inputs.npm_package }}/internal/manifest.json)
+         shell: bash
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ fromJson(steps.manifest.outputs.manifest).commit }}
+          fetch-depth: 0
+      - name: Replace version in package.json
+        run: |
+          package_json="$(jq '.version = "${{ github.event.inputs.version }}"' package.json)" && \
+          echo -E "${package_json}" > package.json    
+      - name: Generate changelog
+        run: npx conventional-changelog-cli@2 -i CHANGELOG.md -s -p conventionalcommits
+      - name: Get number of lines in CHANGELOG.md
+        id: changelog
+        run: echo ::set-output name=changelog_lines::$(wc -l < "CHANGELOG.md")
+        shell: bash
+      - name: Create Release
+        run: gh release create ${{ github.event.inputs.version }} -F CHANGELOG.md --target ${{ fromJson(steps.manifest.outputs.manifest).commit }} --title "Release ${{ github.event.inputs.version }}"
+        if: ${{ steps.changelog.outputs.changelog_lines > 2 }}
```

Comment:
Why are we not making a release it if there are no notes? Having a gap in the version sequence would look like a bug. Any way we can make `conventional-changelog-cli@2` to publish "no customer visible changes in this release"?

Reply:
an empty release will also look like a bug, but I can add that note with bash